client/src/components/data-container.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Graph from "./graph";
+import Table from "./table";
+
+const graphData = [
+    { x: "Monday", y: "point1" },
+    { x: "Tuesday", y: "point2" },
+    { x: "Wednesday", y: "point3" },
+    { x: "Thursday", y: "point4" },
+    { x: "Friday", y: "point5" },
+    { x: "Saturday", y: "point6" },
+    { x: "Sunday", y: "point7" },
+];
+
+const tableData = [
+    { name: "ORP", dataCollected: "Datacollected", percentToDesired: "PTD" },
+    {
+        name: "Temperature",
+        dataCollected: "Datacollected",
+        percentToDesired: "PTD",
+    },
+    { name: "pH", dataCollected: "Datacollected", percentToDesired: "PTD" },
+];
```

Comment:
this is ok, but it is better practice to have this within the component (before the return inside `DataContainer`